# The method getuid is not in the current class but in QtLP_Private.
Index: xvst-2.5/src/qtsingleapplication/src/qtlocalpeer.cpp
===================================================================
--- xvst-2.5.orig/src/qtsingleapplication/src/qtlocalpeer.cpp	2013-05-26 23:46:04.839066000 +0200
+++ xvst-2.5/src/qtsingleapplication/src/qtlocalpeer.cpp	2013-05-27 00:36:10.726976826 +0200
@@ -93,7 +93,7 @@
         socketName += QLatin1Char('-') + QString::number(sessionId, 16);
     }
 #else
-    socketName += QLatin1Char('-') + QString::number(::getuid(), 16);
+    socketName += QLatin1Char('-') + QString::number(QtLP_Private::getuid(), 16);
 #endif
 
     server = new QLocalServer(this);
